.github/workflows/cleanup.yml

```diff
@@ -17,14 +17,20 @@ jobs:
     name: Cleanup
     runs-on: ubuntu-latest-4-cores
     steps:
+      - name: Install werf
+        uses: werf/actions/install@v2
```

Comment:
Let's keep the same order of steps, placing the install step everywhere before the step where we run werf.